pkg/controller/migration/convert/aws-cni-policy-only_test.go

```diff
@@ -192,6 +192,16 @@ func awsCNIPolicyOnlyConfig() []runtime.Object {
 			Spec: appsv1.DeploymentSpec{
 				Template: corev1.PodTemplateSpec{
 					Spec: corev1.PodSpec{
+						Tolerations: []corev1.Toleration{
```

Comment:
I messed up here, actually this Deployment needs to be removed, there is no calico-kube-controllers in the amazon-vpc-cni repo.
This file was meant to represent https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/master/calico.yaml. I think I had copied this and forgot to remove it.

Reply:
I added a commit which removes the kubeController deployment from this test, makes the kubeController in components a pointer, and doesn't throw an error in `getComponents` if it doesn't detect the kubecontrollers deployment :+1: